rclcpp/include/rclcpp/executor.hpp

```diff
@@ -98,6 +214,9 @@ class Executor
 
   /// Remove a node from the executor.
   /**
+   * All the callback groups associated with this node will be removed from the executor
```

Comment:
Including callback groups which are part of the node but were created with the second parameter being false?

Reply:
It should only remove callback groups that are associated with the executor. I changed the comment to reflect this.